docs/_includes/segmented-buttons.html

```diff
@@ -32,6 +32,32 @@ <h3 id="button-default">Default Segmented buttons</h3>
 </div>
     {% endhighlight %}
 
+    <h3 id="button-default">Segmented buttons - large </h3>
+    <div class="demo">
+        <div class="demo__inner">
+            <div class="segmented-buttons segmented-buttons--large">
+                <ul>
+                    <li><button type="button" class="segmented-buttons__button"
+                            aria-current="true">Day</button></li>
```

Comment:
I believe `aria-current` is more appropriate for static implementations, like breadcrumbs. @ianmcburnie and/or @saiponnada can verify this, but in this case, should we not opt to use `aria-selected`?

Reply:
We should address that in another pr/issue. This isn't really relevant since this deals with the size changes. (No scope creep)
But we should get clarity on this offline and file an issue to fix it if so.